src/pages/index.tsx

```diff
@@ -1,22 +1,8 @@
-import React from "react"
-import { Link } from "gatsby"
+import React from 'react';
+import About from '../views/about';
 
-import Layout from "../components/layout"
-import Image from "../components/image"
-import SEO from "../components/seo"
+const index = () => {
+  return <About />;
+};
 
-const IndexPage = () => (
-  <Layout>
-    <SEO title="Home" />
-    <h1>Hi people</h1>
-    <p>Welcome to your new Gatsby site.</p>
-    <p>Now go build something great.</p>
-    <div style={{ maxWidth: `300px`, marginBottom: `1.45rem` }}>
-      <Image />
-    </div>
-    <Link to="/page-2/">Go to page 2</Link> <br />
-    <Link to="/using-typescript/">Go to "Using TypeScript"</Link>
-  </Layout>
-)
-
-export default IndexPage
+export default index;
```

Comment:
Isn't it possible to write just `export { default } from '../views/about'`?

Reply:
good point 👍